apps/betterangels-backend/accounts/schema.py

```diff
@@ -42,9 +44,19 @@
 class Query:
     current_user: UserType = auth.current_user()  # type: ignore
 
-    client_profile: ClientProfileType = strawberry_django.field(
-        extensions=[HasRetvalPerm(perms=[ClientProfilePermissions.VIEW])],
-    )
+    @strawberry_django.field(extensions=[HasRetvalPerm(perms=[ClientProfilePermissions.VIEW])])
+    def client_profile(self, info: Info, pk: strawberry.ID) -> ClientProfileType:
+        client_profile = ClientProfile.objects.prefetch_related(
+            Prefetch(
+                "contacts",
+                queryset=ClientContact.objects.filter(
+                    relationship_to_client=RelationshipTypeEnum.CURRENT_CASE_MANAGER
+                ).order_by("created_at"),
+                to_attr="case_managers",
```

Comment:
nice work!

Reply:
shame that this was necessary. I was hoping the query optimizer would take care of stuff like this